### PR TITLE
Reacting to Hosting changes

### PR DESCRIPTION
**Pull request by JunTaoLuo**
reacting to https://github.com/aspnet/Hosting/pull/445

### CODE REVIEW

File: test/Microsoft.AspNet.SignalR.Server.Tests/Infrastructure/ServiceProviderHelper.cs [hunk 1]
```diff
@@ -33,30 +34,28 @@ public static IServiceProvider CreateServiceProvider(Action<IServiceCollection>
...
-            }
+            private RequestDelegate _requestDelegate;
 
-            public IDisposable Start(IFeatureCollection serverFeatures, Func<IFeatureCollection, Task> application)
+            public IFeatureCollection Features { get; }
+            
+            public void Start(RequestDelegate requestDelegate)
             {
-                return new StartInstance(application);
+                _requestDelegate = requestDelegate;
```
Comment (JunTaoLuo) [inline test/Microsoft.AspNet.SignalR.Server.Tests/Infrastructure/ServiceProviderHelper.cs]: like `_application` of `StartInstance`, this is unused. Maybe remove it?

**Comment by JunTaoLuo** [comment]
updated

**Comment by JunTaoLuo** [comment]
cc @lodejard

**Comment by JunTaoLuo** [comment]
cc @Tratcher

File: test/Microsoft.AspNet.SignalR.Server.Tests/Infrastructure/ServiceProviderHelper.cs (hunk 1 above)
Comment (Tratcher) [inline test/Microsoft.AspNet.SignalR.Server.Tests/Infrastructure/ServiceProviderHelper.cs]: Yeah, just remove it.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**